onnx_tf/handlers/backend/dequantize_linear.py

```diff
@@ -0,0 +1,39 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("DequantizeLinear")
+class DequantizeLinear(BackendHandler):
+
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    x_scale = kwargs["tensor_dict"][node.inputs[1]]
+    x_zero_point = tf.zeros((), x.dtype)
```

Comment:
Don't think tf.zeros is needed. Maybe use numpy zeros? Please also considercombine three lines into one line, like x_zero_point =xxx if cond else yyy.